cli/src/pcluster/aws/aws_resources.py

```diff
@@ -224,10 +224,13 @@ def instance_type(self):
         return self.instance_type_data.get("InstanceType")
 
     def is_cpu_options_supported_in_lt(self):
-        """Check whether hyperthreading can be disabled via CPU options."""
-        # If default threads per core is 1, HT doesn't need to be disabled
+        """Check whether CPUOptions can be set in launch template."""
         # When CpuOptions is not supported, valid threads per core can't be found in vcpu info
-        return self.default_threads_per_core() > 1 and 1 in self.valid_threads_per_core()
+        return len(self.valid_threads_per_core()) != 0
+
+    def is_multi_threading_disablement_supported(self):
```

Comment:
We decided to remove the validator because we have the following cases:
1. DefaultThreadsPerCore > 1 AND ValidThreadsPerCore contains 1 => multi-threading must be disabled via CpuOptions
1. DefaultThreadsPerCore > 1 AND ValidThreadsPerCore not reported => multi-threading must be disabled manually
1. DefaultThreadsPerCore = 1 AND ValidThreadsPerCore contains 1 => nothing to do
1. DefaultThreadsPerCore = 1 AND ValidThreadsPerCore not reported => nothing to do

So there is no case where the validators should fail.